docs/troubleshooting.md

```diff
@@ -16,7 +16,36 @@ If manually running `ollama serve` in a terminal, the logs will be on that termi
 
 Join the [Discord](https://discord.gg/ollama) for help interpreting the logs.
 
-## Known issues
+## LLM libraries
```

Comment:
Would add a note that `OLLAMA_LLM_LIBRARY` is experimental (i.e. folks shouldn't rely on it, rather than using it for debugging). This is the right doc to have it in.